apollo-rx2-support/build.gradle.kts

```diff
@@ -4,8 +4,8 @@ plugins {
 }
 
 java {
-  targetCompatibility = JavaVersion.VERSION_1_7
-  sourceCompatibility = JavaVersion.VERSION_1_7
+  targetCompatibility = JavaVersion.VERSION_1_8
+  sourceCompatibility = JavaVersion.VERSION_1_8
```

Comment:
Should this be factored-in the root `build.gradle.kts` file ? Make sure all sub-projects target JAVA 8 ? Or are Android projects different ?